package.json

```diff
@@ -6,9 +6,10 @@
   "scripts": {
     "test": "cross-env NODE_ENV=test jest --runInBand",
     "buildDB": " cross-env NODE_ENV=development ts-node src/db/build.ts",
+    "build-db": " cross-env NODE_ENV=production ts-node src/db/build.ts",
```

Comment:
this should be
```suggestion
    "build-db": " cross-env NODE_ENV=production node dist/db/build.js",
```

Reply:
The deploy does not work without it

Reply:
make sure your steps are in order so
build and then db build the node version